blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,88 @@
+
+The React Native for Windows team has heard the community feedback. While running on a Windows device will be the best experience for developing on Windows, we recognize that not everyone involved in a development team can have access to multiple devices to develop for different operating systems. We also see that MacBook computers are used widely within the React Native developer community.
+
+**Owning a Windows PC is not a requirement to build and ship Windows experiences using React Native for Windows.** In this blog post, I'm going to walk through a few options for how to develop React Native for Windows applications from your MacBook using several virtual machine options.
+
+To get started building Windows experiences from a non-Windows device, you can leverage virtual machine software. You have several options to spin-up a Windows VM from a Mac including [Parallels](https://www.parallels.com/), [VirtualBox](https://www.virtualbox.org/), [VMWare](https://www.vmware.com/), and [Microsoft Dev Box](https://azure.microsoft.com/products/dev-box/) to name a few. In this post, I'll walk through my experience using Parallels and Microsoft Dev Box to build React Native for Windows applications from a MacBook.
+
+## Creating a Windows VM using Parallels
+
+One option to create VMs on macOS is [Parallels](https://www.parallels.com/), which is software that can be locally installed and supports virtualizing a variety of operating systems including Windows 11.
+
+After signing up for the [Parallels free trial](https://www.parallels.com/products/desktop/trial/), you will be given a download of the Parallels Desktop software and installation instructions. After installing and running Parallels, you will be prompted to install Windows 11. From there,you will be prompted to supply a Windows License Key, which can be purchased from the Microsoft Store. From there, your Windows VM is ready to be run.
+
+With my virtual machine running, I was able to use the VM as if I was on a Windows computer. From here, I could follow the ["System Requirements"](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies) and ["Getting Started"](https://microsoft.github.io/react-native-windows/docs/getting-started) documentation for React Native for Windows. Parallels had access to my local files, so I did not need to redownload any source code to get started developing. When I was finished with my development session, I was able to close the virtual machine and the machine's state was saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-Parallels.jpg)
+
+## Takeaways about Parallels
+
+In less than an hour, I was able to spin up a Windows VM on my Mac and use that environment to build a React Native for Windows app. Windows VMs allowed me to quickly experiment with adding Windows support to my existing React Native app because of the low level of overhead to get a VM up and running.
```

Comment:
```suggestion
Windows VMs via Parallels allowed me to quickly (in less than an hour!) experiment with adding Windows support to my existing React Native app.
```
This is saying the same thing twice. Providing a suggestion for tightening it up.